### PR TITLE
TPD-1611 Delete Giving Pages

### PR DESCRIPTION
**Pull request by jason-rice**
done
[touchpointsoftware.atlassian.net/browse/TPD-1611](https://touchpointsoftware.atlassian.net/browse/TPD-1611)

### CODE REVIEW

File: CmsWeb/src/components/Giving/GivingManagement/GivingPageIndex.vue [hunk 1]
```diff
@@ -1,162 +1,275 @@
...
+}
+.swal-title {
+  font-size: 12px;
+}
+.swal-text {
+  font-size: 12px;
+}
+.delete-button {
+  font-size: 14px;
+}
```
Comment (brondavies) [inline CmsWeb/src/components/Giving/GivingManagement/GivingPageIndex.vue:274]: This is something that should apply the same style everywhere and not just in this component

File: CmsWeb/Areas/Giving/Controllers/GivingManagementController.cs [hunk 2]
```diff
@@ -79,13 +80,12 @@ public ActionResult Create(GivingPageViewModel viewModel)
             return Json(result, JsonRequestBehavior.AllowGet);
         }
 
-        [HttpPost]
```
Comment (brondavies) [inline CmsWeb/Areas/Giving/Controllers/GivingManagementController.cs:82]: We should probably keep the `HttpPost` designation if that's how this is accessed

**Review by brondavies — CHANGES_REQUESTED** [review]

File: CmsWeb/src/components/Giving/GivingManagement/GivingPageIndex.vue (hunk 1 above)
Comment (jason-rice) [reply CmsWeb/src/components/Giving/GivingManagement/GivingPageIndex.vue:274]: deleted this

File: CmsWeb/Areas/Giving/Controllers/GivingManagementController.cs (hunk 2 above)
Comment (jason-rice) [reply CmsWeb/Areas/Giving/Controllers/GivingManagementController.cs:82]: put this attribute back

**Review by josearmandohf — APPROVED** [review]
Changes required were addressed.

**Pull request merged**